src/test/java/org/jenkinsci/plugins/gitclient/GitClientTest.java

```diff
@@ -90,6 +91,9 @@
     /* Instance of object under test */
     private GitClient gitClient = null;
 
+    /* Instance of object of another test class*/
+    private CliGitAPIImplTest cliGitAPIImplTest = new CliGitAPIImplTest();
```

Comment:
I would prefer to move the definition of the VersionTest inner class from the CliGitAPIImplTest class to this class rather than reference it here.  Eventually we hope to remove the CliGitAPIImplTest class completely so that it spends no time executing tests in that file.